File: tests/fixtures/coding-eval/python/sample.py
LABEL_PREFIX = "item"


def find_user(users, user_id):
    return next((user for user in users if user["id"] == user_id), None)


def divide(left, right):
    return left // right


def label(name):
    return f"{LABEL_PREFIX}:{name}"


def normalize_name(name):
    return " ".join(name.strip().lower().split())


def active(status):
    return status == "active"


def is_admin(role):
    return role is "admin"

File: tests/fixtures/coding-eval/java/src/Sample.java
import java.util.List;
import java.util.Map;
import java.util.Optional;

public final class Sample {
    private static final String LABEL_PREFIX = "item";

    public record User(int id, String name) {}

    private Sample() {}

    public static Optional<User> findUser(List<User> users, int userId) {
        return users.stream().filter(user -> user.id() == userId).findFirst();
    }

    public static double divide(int left, int right) {
        return left / right;
    }

    public static String label(String name) {
        return LABEL_PREFIX + ":" + name;
    }

    public static String normalizeName(String name) {
        return String.join(" ", name.strip().toLowerCase().split("\\s+"));
    }

    public static boolean active(Map<String, String> status) {
        return status.equals("active");
    }

    public static boolean isAdmin(String role) {
        return role.startsWith("admin");
    }
}
